requirements.txt

```diff
@@ -3,8 +3,9 @@ apache_beam
 earthengine-api
 geopandas
 google_apitools
-google-cloud-aiplatform
+google-cloud-aiplatform==1.26.1
```

Comment:
Can you elaborate on why it's necessary to peg the requirement to a specific version?